DNN Platform/Library/Entities/Portals/PortalController.cs

```diff
@@ -2340,42 +2285,6 @@ private static void ParsePortalDesktopModules(XPathNavigator nav, int portalID)
             }
         }
 
-        private static void UpdatePortalSettingInternal(int portalID, string settingName, string settingValue, bool clearCache, string cultureCode, bool isSecure)
-        {
-            string currentSetting = GetPortalSetting(settingName, portalID, string.Empty, cultureCode);
-
-            if (currentSetting != settingValue)

```

Comment:
Is this check for the new value being equal to the old value still in the new implementation?  I don't remember seeing something like that.